src/llvm.nim

```diff
@@ -29,26 +32,30 @@ type
 
 # Import the matching version:
 const
-  LLVMV* {. strdefine .} = "5.0"
-  LLVMLib* {. strdefine .} = "libLLVM-" & LLVMV & ".so"
+  LLVMV* {. intdefine .} = 11
```

Comment:
Older versions of LLVM used non-int versions (e.g. `3.7`). This change breaks them. I think a string should still be accepted, and then converted to a float inside this file.

Reply:
> Thanks for the PR! It's pretty different from the other versions, though. Did you use `build.sh`? If so, I suppose it is `c2nim` that generates different code now.

I'm not a fan of c2nim, I just made a copy of version 5.0 and then made changes by myself